multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/CollectServicesInProgressStateStep.java

```diff
@@ -81,4 +81,9 @@ protected String getStepErrorMessage(ProcessContext context) {
         return Messages.ERROR_MONITORING_OPERATIONS_OVER_SERVICES;
     }
 
+    @Override
+    public Duration getTimeout(ProcessContext context) {
+        return context.getVariable(Variables.START_TIMEOUT);
```

Comment:
I'd suggest creating a new variable for this step instead of using the START_TIMEOUT, it is already used improperly and soon will be removed